felix/fv/infrastructure/etcd.go

```diff
@@ -15,24 +15,38 @@
 package infrastructure
 
 import (
+	"fmt"
+	"os"
 	log "github.com/sirupsen/logrus"
 
 	"github.com/projectcalico/calico/felix/fv/containers"
 	"github.com/projectcalico/calico/felix/fv/utils"
 )
 
 func RunEtcd() *containers.Container {
+	args := []string{
```

Comment:
You'll need to run `goimports` on this PR as it looks like the formatting isn't quite right.

Reply:
OK, I will update to it.